### DISCUSSION TITLE
Change color of cursor according to mode

### DISCUSSION THREAD

**Discussion by fvacek** [helix-editor/helix#7397] [Ideas] [answered]
Mode indicator color is changed according to mode used. I thing that also cursor color and possibly also cursor line color can vary to inform user about current mode without looking to bottom left corner.

**Comment by the-mikedavis** [comment, marked as answer]
You can theme cursors by mode with the `ui.cursor.{normal,insert,select}` theme keys and the primary cursor with `ui.cursor.primary.{normal,insert,select}`: https://docs.helix-editor.com/master/themes.html#interface
